include/dump_offload.hpp

```diff
@@ -351,8 +374,8 @@ inline void requestRoutes(App& app)
 
             boost::asio::io_context* ioCon = conn.getIoContext();
 
-            std::string unixSocketPath =
-                unixSocketPathDir + dumpType + "_dump_" + dumpId;
+            std::string unixSocketPath = unixSocketPathDir + dumpType +
```

Comment:
Please add a comment to say that you are using rand() to create different unique sockets for each dump offload request

Reply:
done